source/pt-br/docs/variables.md

```diff
@@ -1,93 +1,94 @@
 `path` | Path of current page | `string`
-`url` | Full URL of current page | `string`
-`env` | Environment variables | ???
+`url` | URL completo da página atual | `string`
+`env` | Variáveis de ambiente | ???
 
-### Site Variables
+### Variáveis do site
 
-Variable | Description | Type
+Variáveis | Descrição | Tipo
 --- | --- | ---
-`site.posts` | All posts | `array` of `post` objects
-`site.pages` | All pages | `array` of `page` objects
-`site.categories` | All categories | `array` of ???
-`site.tags` | All tags | `array` of ???
+`site.posts` | Todos os posts | `array` de `post` objetos
+`site.pages` | Todas as páginas | `array` de `page` objetos
+`site.categories` | Todas as categorias | `array` de ???
+`site.tags` | Todas as tags | `array` de ???
 
-### Page Variables
+### Variáveis da página
 
-**Article (`page`)**
+**Artigo (`page`)**
 
-Variable | Description | Type
+Variáveis | Descrição | Tipo
 --- | --- | ---
-`page.title` | Article title | `string`
-`page.date` | Article created date | [Moment.js] object
-`page.updated` | Article last updated date | [Moment.js] object
-`page.comments` | Comment enabled or not | `boolean`
-`page.layout` | Layout name | `string`
-`page.content` | The full processed content of the article | `string`
-`page.excerpt` | Article excerpt | `string`
-`page.more` | Contents except article excerpt | `string`
-`page.source` | The path of the source file | `string`
-`page.full_source` | Full path of the source file | `string`
-`page.path` | The URL of the article without root URL. We usually use `url_for(page.path)` in theme. | `string`
-`page.permalink` | Full URL of the article | `string`
-`page.prev` | The previous post, `null` if the post is the first post | ???
-`page.next` | The next post, `null` if the post is the last post | ???
-`page.raw` | The raw data of the article | ???
-`page.photos` | The photos of the article (Used in gallery posts) | array of ???
-`page.link` | The external link of the article (Used in link posts) | `string`
-
-**Post (`post`):** Same as `page` layout but add the following variables.
-
-Variable | Description | Type
+`page.title` | Artigo title | `string`
+`page.date` | Artigo criado data | [Moment.js] objeto
+`page.updated` | Última data atualizada do artigo | [Moment.js] object
+`page.comments` | CComentário habilitado ou não | `boolean`
+`page.layout` | Layout nome | `string`
+`page.content` | O conteúdo completo processado do artigo | `string`
+`page.excerpt` | Trecho do artigo| `string`
+`page.more` | Conteúdo exceto trecho do artigo | `string`
+`page.source` | O caminho do arquivo de origem | `string`
+`page.full_source` | Caminho completo do arquivo de origem | `string`
+`page.path` | O URL do artigo sem URL de raiz. Usamos geralmente `url_for(page.path)` no tema. | `string`
+`page.permalink` | URL completo do artigo | `string`
+`page.prev` | O post anterior, `null` se o post for o primeiro post | ???
```

Comment:
Creio que assim ficaria melhor:  'O post anterior, `null` se for o primeiro post'